commands/mr/delete/mr_delete.go

```diff
@@ -18,7 +19,11 @@ func NewCmdDelete(f *cmdutils.Factory) *cobra.Command {
 		Long:    ``,
 		Args:    cobra.MaximumNArgs(1),
 		Aliases: []string{"del"},
-		Example: "$ glab delete 123",
+		Example: heredoc.Doc(`
+			$ glab delete 123
+			$ glab del 123
+			$ glab delete branch
```

Comment:
You missed the `mr` command here:
```suggestion
			$ glab mr delete 
			#=> deletes mr for current branch

			$ glab mr delete 123
			$ glab mr del 123
			$ glab mr delete branch
```